ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/platform.md

```diff
@@ -122,4 +132,38 @@
 
         # Maximum number of CPUs used for data plane traffic forwarding.
         data_plane_cpu_allocation_max: <int; 1-128>
+
+        # Configure interface related settings for Sfe platform.
+        interface:
+
+          # Configure one or more Receive Side Scaling (RSS) interface profiles.
+          # This is supported on select platforms.
+          profiles:
```

Comment:
Are there other `interface xxx` options or could we collapse this to `interface_profiles` ?

Reply:
nevermind you have interface_profile under it as well.